### PR TITLE
Document Electionpledge.org

### PR DESCRIPTION
**Pull request by BerylBogui**
Document Electionpledge.org (EU) 
Place it in a new "prevention" section

### CODE REVIEW

File: _pages/encyclopedia/Preventing disinformation.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The Commission is part of the **Campaign for Democracy program**, which “_aims to become the world’s leading eco-system of dissidents and supporters of democracy_” by “_engaging supporters of democracy worldwide and building a powerful intellectual movement_”.
+
+This Campaign was itself launched by the [**Alliance of Democracies Foundation**](http://www.allianceofdemocracies.org/),  a non-profit organization founded by Anders Fogh Rasmussen, the former NATO Secretary General and former Prime Minister of Denmark in 2017. Its core mission is “_the advancement of democracy and free markets across the globe_”.
+
+### What is the Pledge?
+
+The Pledge for Election Integrity is a call from the TCEI to candidates for the upcoming European elections in May 2019. They urge candidates “_to take a stand against election interference by increasing transparency and fight the use of disinformation in campaigns_”.
+
+The Pledge was intentionally launched in a tense, heavy electoral context. Indeed, in the next year or so (2019-2020), major elections will take place in the European Union, the US and Canada.
```
Comment (MattiSG) [inline _pages/encyclopedia/Preventing disinformation.md]: When isn't this the case exactly? 🤔

File: _pages/encyclopedia/Preventing disinformation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+The Commission is part of the **Campaign for Democracy program**, which “_aims to become the world’s leading eco-system of dissidents and supporters of democracy_” by “_engaging supporters of democracy worldwide and building a powerful intellectual movement_”.
+
+This Campaign was itself launched by the [**Alliance of Democracies Foundation**](http://www.allianceofdemocracies.org/),  a non-profit organization founded by Anders Fogh Rasmussen, the former NATO Secretary General and former Prime Minister of Denmark in 2017. Its core mission is “_the advancement of democracy and free markets across the globe_”.
+
+### What is the Pledge?
+
+The Pledge for Election Integrity is a call from the TCEI to candidates for the upcoming European elections in May 2019. They urge candidates “_to take a stand against election interference by increasing transparency and fight the use of disinformation in campaigns_”.
```
Comment (MattiSG) [inline _pages/encyclopedia/Preventing disinformation.md]: This should probably be the first thing the reader sees. History and origins are quite secondary.

File: _pages/encyclopedia/Preventing disinformation.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The Pledge was intentionally launched in a tense, heavy electoral context. Indeed, in the next year or so (2019-2020), major elections will take place in the European Union, the US and Canada. 
+
+### Engagements
+
+Below are the five core engagements of the pledge as listed on the official website. Signatories are:
```
Comment (MattiSG) [inline _pages/encyclopedia/Preventing disinformation.md]: Do we really need to copy the content here? 😉 Let's just summarize and give a link.

File: _pages/encyclopedia/Preventing disinformation.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+4. Taking active steps to maintain **cyber security** and to train campaign staff in **media literacy and risk awareness** to recognise and prevent attacks;
+
+5. Committing to transparency about the sources of campaign **finances**.
+
+### Who signs the Pledge?
+
+Whether they be political parties or candidates seeking office, “_anyone that is or intends to run as a candidate for the European Parliament elections in May 2019, the U.S. elections in November 2020, or the Canadian parliamentary in October 2019 may take the pledge._”
+
+As of May 09th, 2019, 10 candidates have taken the pledge in the European Union, from Germany, the Netherlands, Czech Republic, Belgium, Romania and the U.K. (Scotland).
```
Comment (MattiSG) [inline _pages/encyclopedia/Preventing disinformation.md]: Such information will always be outdated. Do not include it, give as direct links as possible instead.

File: _pages/encyclopedia/Preventing disinformation.md (hunk 1 above)
Comment (BerylBogui) [reply _pages/encyclopedia/Preventing disinformation.md]: Well it was highlighted several times and it is a transatlantic initiative so I think it's important to underline it :/

File: _pages/encyclopedia/Preventing disinformation.md (hunk 4 above)
Comment (BerylBogui) [reply _pages/encyclopedia/Preventing disinformation.md]: Shouldn't we leave it to show the situation right before the elections though? It's a measure of the importance of the pledge I think

Comment (MattiSG) [reply _pages/encyclopedia/Preventing disinformation.md]: Okay, that's a good point 🙂 I had not taken into account this notion of measuring the potential impact on the EU elections 👍
I would then recommend to:
1. Commit to update this piece of content right before the elections.
2. Put that number into perspective with the total number of candidates.

**Pull request merged**